packages/python-packages/apiview-copilot/README.md

```diff
@@ -17,37 +17,31 @@ AZURE_COSMOS_DB_NAME="..."  # The name of the CosmosDB database
 ```
 
 3. Create one or more test files in plain-text for the language of choice. Store them in `scratch/apiviews/<lang>/`.
-4. Generate a review using `python cli.py review generate --language <lang> --target <path_to_target_file> [--base <path_to_base_file>] [--debug-log]`.
+4. Generate a review using `python cli.py review generate --language <lang> --target <path_to_target_file> [--base <path_to_base_file>] [--debug-log] [--remote]`.
 5. Examine the output under `scratch/output/<lang>/<test_file>.json`.
 
 ## Review Process and Stages
 
 For each section of the APIView, the review process now consists of three distinct stages:
 
-- **Guideline Stage:** Reviews the section against language-specific guidelines retrieved from Azure Search and CosmosDB.
+- **Guideline Stage:** Reviews the section against language-specific guidelines.
 - **Context Stage:** Reviews the section using the full context (guidelines, examples, and memories) retrieved for that section.
 - **Generic Stage:** Applies generic review rules and best practices.
 
-All context and guidelines are retrieved dynamically from Azure Search and CosmosDB. There is no static guideline mode.
-
-### Example: Generating a Review
+## Creating Reviews
 
 ```
-python cli.py review generate --language <lang> --target <path_to_target_file> [--base <path_to_base_file>] [--debug-log]
+python cli.py review generate --language <lang> --target <path_to_target_file> [--base <path_to_base_file>] [--debug-log] [--remote]
 ```
 
-- Use `--debug-log` to dump kept and discarded comments to files for debugging purposes.
-
-## Creating Reviews with Development Code
-
-- `cli.bat review local`: Generate a review using the development code. Supports the `--debug-log` option.
+- Use `--debug-log` to dump kept and discarded comments to files for debugging purposes. Only supported when calls are made locally.
+- Use `--remote` to generate a review using the deployed Copilot app rather than making local calls.
 
 ## Flask App in App Service
 
 Commands available for working with the Flask app:
 
 - `cli.bat app deploy`: Deploy the Flask app to Azure App Service.
```

Comment:
Running cli.bat in Linux doesn't work for me. Is this only for Windows or does anything else need to be done to run it?